### DISCUSSION TITLE
Add multiple markers

### DISCUSSION THREAD

**Discussion by gabidusa** [liodali/osm_flutter#402] [Q&A]
Hello,

Is there a way to add multiple markers on a map?

Currently the way I found to do this is by calling `await mapController.addMarker(point, markerIcon: const MarkerIcon(icon: Icon(Icons.add)), angle: pi / 3);` for each market that I have. 
But this comes with a performance issue as you can see the markers added on the map.
For example, for 50 markers it takes 30-40 seconds to be rendered on the map.

Thanks!

**Comment by liodali** [comment]
we have static position api if yoi have lot of markers to illistrate some station or similiar behavior
